plugins/node/opentelemetry-instrumentation-fastify/src/version.ts

```diff
@@ -1,18 +0,0 @@
-/*
```

Comment:
Good catch but this might be better done in a separate PR since it has nothing to do really with what the PR is about.